ff/serving/metrics/interface.go

```diff
@@ -126,6 +127,22 @@ func (p PromFeatureObserver) Finish() {
 	p.Count.WithLabelValues(p.Name, p.Feature, p.Key, "success").Inc()
 }
 
+func (p PromFeatureObserver) GetObservedRowCount() (int, error) {
+	var m = &dto.Metric{}
+	if err := p.Count.WithLabelValues(p.Name, p.Feature, p.Key, "row serving").Write(m); err != nil {
```

Comment:
Should the parameter that "row serving" is passed to accept an enum instead of a plain string?

Reply:
yes, in fact since it is queried from the front end with it as an explicit label value is should be defined once